install-storageos/030-install-storageos-cli.md

```diff
@@ -0,0 +1,22 @@
+StorageOS has a CLI and GUI. We are using the CLI to verify the installation.
```

Comment:
Let's mention that for convenience, we run it in a pod and access using kubectl exec.